Jenkinsfile

```diff
@@ -202,8 +202,8 @@ stage('Unit Test') {
         unpack_lib('gpu', tvm_multilib)
         timeout(time: max_time, unit: 'MINUTES') {
           sh "${docker_run} ${ci_gpu} ./tests/scripts/task_sphinx_precheck.sh"
-          sh "${docker_run} ${ci_gpu} ./tests/scripts/task_python_unittest.sh"
-          sh "${docker_run} ${ci_gpu} ./tests/scripts/task_python_integration.sh"
+          sh "${docker_run} ${ci_gpu} ./tests/scripts/task_python_unittest.sh gpu"
```

Comment:
Let us consider add new scripts e.g. `tests/scripts/task_python_unittest_gpuonly.sh` that setup the scripts and runs runs `task_python_unit_test.sh` . This way devs can always directly go and run the scripts

Reply:
Without any arguments, the scripts should be the same as before this PR. The gpu or cpu arguments are completely optional.